appveyor.yml

```diff
@@ -58,6 +58,13 @@ install:
     - cmd: set PATH=%PATH%;%CONDA_INSTALL_LOCN%;%CONDA_INSTALL_LOCN%\scripts
     - cmd: set PYTHONUNBUFFERED=1
 
+    - cmd: conda list
+    - cmd: conda install --yes ruamel_yaml
+    - cmd: conda list
+    - cmd: conda install --yes --force python=3.5
```

Comment:
`conda-build-all` needs to be updated to work with Py36. Force-downgrading python to Py35 introduces other problems into the builds.